src/main/scala/com/gd/twitteranalytics/util/TweetsConstant.scala

```diff
@@ -0,0 +1,11 @@
+package com.gd.twitteranalytics.util
+
+object TweetsConstant {
+
+  val PARTITION_COLUMN = "event_date"
+  val SAVEPATH = "twitter-analytics/dataLake/eng-tweets"
+  val TABLENAME = "engtweets"
+  val OUTPUT_DATE_FORMAT = "YYYYMMdd HH.mm.ss"
```

Comment:
up-to-seconds granularity is unpractical for the majority of workloads (including Tweeter use-case) 
- too many directories/small files can overload Namenone index and fragment storage (http://blog.cloudera.com/blog/2009/02/the-small-files-problem/)

Please organize data by `year-month-day/hour-minute`.

Reply:
Done.

Reply:
don't see a commit with the change. the latest one was published on 24/09.